cetmix_tower_server/tests/test_plan.py

```diff
@@ -1583,6 +1583,8 @@ def test_plan_line_action_variable_values_user_access_rights(self):
         #  plan line action
         # Add user to group
         self.add_to_group(self.user_bob, "cetmix_tower_server.group_user")
+        self.variable_os.write({"access_level": "1"})
```

Comment:
You should use the helper [write_and_invalidate](https://github.com/cetmix/cetmix-tower/blob/dbceb79bd45e80a0436632f2c383182619eda61b/cetmix_tower_server/tests/common.py#L362) method to ensure that record is updated properly. Check how it used in other places in this file, eg [here](https://github.com/cetmix/cetmix-tower/blob/dbceb79bd45e80a0436632f2c383182619eda61b/cetmix_tower_server/tests/test_plan.py#L1601).